src/pages/git/git-pull/index.md

```diff
@@ -3,13 +3,31 @@ title: Git Pull
 ---
 ## Git Pull
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/git/git-pull/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+[**git pull**](https://git-scm.com/docs/git-pull) allows you to get new commits from a remote repository. To do this, *git pull* fetches ( *git fetch* ) the new commits and merges [(*git merge*)](https://guide.freecodecamp.org/git/git-merge) these into your local branch.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+This command's syntax is as follows:
+```bash
+git pull [options] [<repository>]
```

Comment:
We had an older issue with using the angle brackets even in code-fenced examples (they were incorrectly parsed/rendered as HTML, and the article looked weird), so we're using a different all-caps-with-hyphens format to avoid them entirely. This command will look a little goofy, but to be consistent with the other Git articles, change this to `git pull OPTIONS REPOSITORY`. Also, there's a bullet about `refspec` below - should there be a placeholder for it here to avoid confusion?